### PR TITLE
XE2 and XE3

### PR DESCRIPTION
**Pull request by andre-m-dev**
Foundation for XE2 and XE3 (Inspired by https://github.com/andrew-svirin/ebics-client-php/blob/2.x/src/Factories/RequestFactoryV25.php#L687)

Tests with CH pain files are still missing, but I'm not sure where to get them yet

### CODE REVIEW

**Comment by andre-m-dev** [comment]
I am not one hundred percent satisfied and I would not put my hand in the fire as to whether this will work with every bank in Switzerland. 🙈

**Comment by tobischo** [comment]
I doubt that we will be able to verify it for every bank in switzerland, so there isn't much we can do about that.

If people use it and notices that something isn't working for a particular bank, they can open an issue with a bug report.

Did you manage to test/verify it for your target bank though?

File: lib/epics/xe2.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Epics::XE2 < Epics::CDD
```
Comment (tobischo) [inline lib/epics/xe2.rb:1]: if we go for inheritance here, I think XE2 should inherit from CCT, given that it is uploading pain.001 files

**Review by tobischo — CHANGES_REQUESTED** [review]

File: lib/epics/xe2.rb (hunk 1 above)
Comment (andre-m-dev) [reply lib/epics/xe2.rb:1]: For sure, good point! I will change it

Comment (andre-m-dev) [reply lib/epics/xe2.rb]: Changed

**Comment by tobischo** [comment]
Alright, so I verified it against the test environment of credit suisse. It at least accepted uploading the files through the order types XE2 and XE3

**Review by tobischo — APPROVED** [review]

**Comment by andre-m-dev** [comment]
@tobischo thank you :pray:

**Pull request merged**